macros/models/_sector/nft/platforms/seaport_v3_trades.sql

```diff
@@ -0,0 +1,724 @@
+                  ,'match_ord' as sub_type
+                  ,execution_idx as sub_idx
+                  ,from_hex(json_extract_scalar(json_extract_scalar(orders[1],'$.parameters'),'$.zone')) as zone
+                  ,from_hex(json_extract_scalar(json_extract_scalar(orders[1],'$.parameters'),'$.offerer')) as offerer
+                  ,json_extract_scalar(json_extract_scalar(json_extract_scalar(orders[1],'$.parameters'),'$.offer[0]'),'$.itemType') as offer_first_item
+                  ,json_extract_scalar(json_extract_scalar(json_extract_scalar(orders[1],'$.parameters'),'$.consideration[0]'),'$.itemType') as consider_first_item
+                  ,from_hex(json_extract_scalar(execution,'$.offerer')) as sender
+                  ,from_hex(json_extract_scalar(json_extract_scalar(execution,'$.item'),'$.token')) as token_contract_address
+                  ,json_extract_scalar(json_extract_scalar(execution,'$.item'),'$.amount') as original_amount
+                  ,json_extract_scalar(json_extract_scalar(execution,'$.item'),'$.itemType') as item_type_code
+                  ,cast(json_extract_scalar(json_extract_scalar(execution,'$.item'),'$.identifier') as uint256) as token_id
+                  ,from_hex(json_extract_scalar(json_extract_scalar(execution,'$.item'),'$.recipient')) as receiver
+                  ,contract_address as platform_contract_address
+            from (select *
+                    from {{ Seaport_call_matchOrders }}
+                    cross join unnest(output_executions) with ordinality as foo(execution,execution_idx)
+                   where call_success
+                     and contract_address = 0x00000000006c3852cbef3e08e8df289169ede581  -- Seaport v1.1
+                 {% if not is_incremental() %}
+                     and call_block_time >= timestamp '{{start_date}}'  -- seaport first txn
+                  {% else %}
+                     and call_block_time >= date_trunc('day', now() - interval '7' day)
+                 {% endif %}
+                )
+    )
+)
+,iv_base_pairs as (
+    select a.block_time
+            ,a.block_number
+            ,a.tx_hash
+            ,a.evt_index
+            ,a.sub_type
+            ,a.sub_idx
+            ,a.offer_first_item_type
+            ,a.consideration_first_item_type
+            ,a.offerer
+            ,a.recipient
+            ,a.sender
+            ,a.receiver
+            ,a.zone
+            ,a.token_contract_address
+            ,a.original_amount
+            ,a.item_type
+            ,a.token_id
+            ,a.platform_contract_address
+            ,a.offer_cnt
+            ,a.consideration_cnt
+            ,a.order_hash
+            ,a.is_private
+            ,a.is_self_trans
+            ,a.is_platform_fee
+            ,a.eth_erc_idx
+            ,a.nft_cnt
+            ,a.erc721_cnt
+            ,a.erc1155_cnt
+            ,date_trunc('day', a.block_time) as block_date
```

Comment:
minor: cast this to date? otherwise, it'll pull through timestamp type from `block_time`

Reply:
(i'll only comment on this instance)

Reply:
There's a lot of these still lingering in the NFT dependency tree.
They should indeed all be cast to date. :+1: